pytest.ini

```diff
@@ -0,0 +1,6 @@
+[pytest]
+filterwarnings =
+    ignore::DeprecationWarning
+    ignore::PendingDeprecationWarning
+markers =
+    pep8: workaround for https://bitbucket.org/pytest-dev/pytest-pep8/issues/23/
```

Comment:
Please add the missing newline at the end of the file

Reply:
Added